src/containers/Cech.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
You should rename this file from `Cech.jsx` to `CECHProject.jsx`. File names should match the name of the component they contain